custom/icds_reports/tests/test_export_data.py

```diff
@@ -964,91 +964,91 @@ def test_awc_infrastructure_export(self):
                     [
                         [
                             "State",
-                            "Percentage AWCs with drinking water",
-                            "Percentage AWCs with functional toilet",
-                            "Percentage AWCs with medicine kit",
-                            "Percentage AWCs with weighing scale: infants",
-                            "Percentage AWCs with weighing scale: mother and child"
+                            "Percentage AWCs reported clean drinking water",
+                            "Percentage AWCs reported functional toilet",
+                            "Percentage AWCs reported medicine kit",
+                            "Percentage AWCs reported weighing scale: infants",
+                            "Percentage AWCs reported weighing scale: mother and child"
                         ],
                         [
                             "st1",
-                            "50.00 %",
-                            "25.00 %",
-                            "30.77 %",
-                            "38.46 %",
-                            "13.46 %"
+                            "2600.00 %",
```

Comment:
Why are these numbers so high?

Reply:
@emord I checked in output data for tests and a lot of records have field ```infa_last_update_date != null``` so we have a situation where nominator > denominator.